machine/dni/dni_3048up/INSTALL

```diff
@@ -0,0 +1,92 @@
+============================================================
+Installing ONIE on DNI 3048UP
+============================================================
+
+Cross-Compiling ONIE
+====================
+
+Change directories to ``build-config`` to compile ONIE.
+
+To compile ONIE first change directories to ``build-config`` and then
+type ``"make MACHINEROOT=../machine/dni/ MACHINE=dni_3048up all"``.
+For example::
+
+  $ cd build-config
+  $ make -j4 MACHINEROOT=../machine/dni/ MACHINE=dni_3048up all
+
+When complete, the ONIE binaries are located in
+``build/images``::
+
+  -rw-r--r-- 3604480 May  7 11:38 onie-dni_3048up-r1.bin
```

Comment:
Looking at the .dts file, the `uboot_ubootenv-up` format looks correct for this platform.  There is no need to modify the script.

The ${output_bin} and ${output_bin}.uboot+env files are only used to install ONIE from the u-boot prompt following the platform's INSTALL instructions.  MTD partitions are not used at the u-boot prompt.

These files are not used during subsequent ONIE updates, i.e. when using ONIE to update ONIE, where the mtd partitions matter.  The onie-self-update image does use MTD partitions, but the raw images are used in that case.

Are you able to install these files following the platform's INSTALL instructions?